readthedocs/ssh/models.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import division, print_function, unicode_literals
+
+import base64
+import hashlib
+import json
+
+from builtins import zip
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
+
+from readthedocs.projects.models import Project
+
+from .mixins import SSHKeyGenMixin
+from .querysets import SSHKeyQuerySet
+
+
+@python_2_unicode_compatible
+class SSHKey(SSHKeyGenMixin, models.Model):
+
+    pub_date = models.DateTimeField(_('Publication date'), auto_now_add=True)
```

Comment:
Nitpick: `pub_date` implies 'publication date' of an article, `create_date` makes more sense

Reply:
I don't like it either but I just wanted to keep coherence with the rest of the models.

Project, RemoteOrganization, RemoteRepositoy and GoldUser use `pub_date` (and also many models from corporate)

Reply:
We also use `create_date` and others though, so we can avoid following the bad pattern here